src/libmongoc/tests/test-mongoc-topology-reconcile.c

```diff
@@ -456,13 +447,12 @@ test_topology_reconcile_from_handshake (void *ctx)
  * mode for good measure.
  */
 static void
-_test_topology_reconcile_retire (bool pooled)
+test_topology_reconcile_retire_single (void)
```

Comment:
The single threaded scanner has a concept of "retired" nodes, for nodes removed in the middle of a full scan. There is no equivalent for multi-threaded, so this test is now limited to single-threaded.